components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -794,7 +791,7 @@ void AdsServiceImpl::CheckIdleState() {
 }
 
 void AdsServiceImpl::ProcessIdleState(const ui::IdleState idle_state,
```

Comment:
`const ui::IdleState idle_state` is an enum. Doesn't need const?

Reply:
The automated script (see Slack), missed this because it does not have the `Type` suffix. I will manually fix this now :-), thanks